esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -2815,6 +2815,9 @@ bool CFileSprayEx::onFileList(IEspContext &context, IEspFileListRequest &req, IE
                 throw MakeStringException(ECLWATCH_ACCESS_TO_FILE_DENIED, "Only cfg or log file allowed.");
         }
 
+        if (!validateDropZonePath(nullptr, netaddr, sPath) && !validateConfigurationDirectory(nullptr, "log", nullptr, nullptr, sPath)) //The path should be the absolute path for the dropzone or log file.
```

Comment:
I know this has generally been the case, but requiring an absolute name for the dropzone or log path seems wrong - since it is exposing internal information that shouldn't be required.  Should probably be revisited in a separate PR